scanner/src/lib/dataproducts/battery/ManufacturingDataSheet_v0.1.svelte

```diff
@@ -0,0 +1,153 @@
+    const result = s.replace(/([A-Z])/g, " $1")
+    return result.charAt(0).toUpperCase() + result.slice(1)
+  }
+</script>
+
+<article>
+  <DataRow label="Product Name" value={data.productName} />
+  <DataRow label="Battery Model" value={data.batteryModel} />
+  <DataRow label="Battery Category" value={data.batteryCategory} />
+  <DataRow label="Manufacturing Date" value={data.manufacturingDate} />
+  <DataRow label="Weight" value={formatNumber(data.weight, "kg")} />
+  <DataRow label="Capacity" value={formatNumber(data.capacity, "Ah")} />
+  <DataRow label="Power" value={formatNumber(data.power, "Wh")} />
+  <DataRow label="Extinguishing Agents" value={data.extinguishingAgents.join(", ")} />
+  <DataRow label="Warranty" value={data.warranty} />
+  <div class="divider" />
+  <div class="title no-bottom-margin">Manufacturer Location</div>
+  <div class="subtitle">The details of the battery manufacturer</div>
+  {#each Object.entries(data.manufacturerInformation) as [key, value]}
+    <DataRow label={camelCaseToWords(key)} {value} />
+  {/each}
+  <div class="divider" />
+  <div class="title no-bottom-margin">Manufacturing Location</div>
+  <div class="subtitle">The details of the location of the battery manufacturing plant</div>
+  {#each Object.entries(data.manufacturingLocation) as [key, value]}
+    <DataRow label={camelCaseToWords(key)} {value} />
+  {/each}
+  <div class="divider" />
+  <div class="title no-bottom-margin">Round Trip Efficiency</div>
+  <div class="subtitle">The details of the round trip energy efficiency in energy storages</div>
+  {#each Object.entries(data.roundTripEfficiency) as [key, value]}
+    <DataRow label={camelCaseToWords(key)} value={formatNumber(value, "%")} />
+  {/each}
+  <div class="divider" />
+  <div class="title no-bottom-margin">Voltage Levels</div>
+  <div class="subtitle">The details of the voltage levels of the battery</div>
+  {#each Object.entries(data.voltageLevels) as [key, value]}
+    <DataRow label={camelCaseToWords(key)} value={formatNumber(value, "V")} />
+  {/each}
+  <div class="divider" />
+  <div class="title no-bottom-margin">Temperature Range</div>
+  <div class="subtitle">The details of the acceptable temperature values of the battery</div>
+  {#each Object.entries(data.temperatureRange) as [key, value]}
+    <DataRow label={camelCaseToWords(key)} value={formatNumber(value, "C°")} />
+  {/each}
+  <div class="divider" />
+  <div class="title no-bottom-margin">Expected Lifetime</div>
+  <div class="subtitle">The details of the battery lifetime</div>
+  <DataRow label="Cycle Life" value={formatNumber(data.expectedLifetime.cycleLife)} />
+  <DataRow label="Reference Test" value={data.expectedLifetime.referenceTest} />
+  <DataRow label="Cycle Rate" value={data.expectedLifetime.cycleRate} />
+  <div class="divider" />
+  <div class="title no-bottom-margin">Material Composition</div>
+  <div class="subtitle">The details of the material composition of the battery</div>
```

Comment:
So where are the details of the battery composition?

![image](https://github.com/ioxiocom/ioxio-tags/assets/1296558/c71f1bad-3e44-4754-8c9d-344ec62343f8)

![image](https://github.com/ioxiocom/ioxio-tags/assets/1296558/5636e690-9b0f-460f-898f-548337ddc0cb)

The design for this section is also very obviously wrong so it cannot be implemented as-is